statsmodels/stats/tests/test_contingency_tables.py

```diff
@@ -61,7 +61,8 @@ def test_SquareTable_from_data():
                  rslt3.summary().as_text())
 
     s = str(rslt1)
-    assert_equal(s.startswith('A 5x5 contingency table with counts:\n[[ 8.'), True)
+    assert_equal(s.startswith('A 5x5 contingency table with counts:'), True)
```

Comment:
numpy 1.14.0 seems to remove the leading space between the `\n[[` and the `8.`. I'm just going to check the text matches and the first element of the 2 dimensional array matches.